<launch>
	<arg name="on_decription" default="true" />
	<arg name="fixed_frame_name" default="rslidar" />
	<arg name="voxel_leafsize" default="0.10"/>
	<arg name="x_max" default="3.0"/>
	<arg name="x_min" default="-3.0"/>
	<arg name="y_max" default="3.0"/>
	<arg name="y_min" default="-3.0"/>
	<arg name="z_max" default="3.0"/>
	<arg name="z_min" default="-3.0"/>
	<node pkg="preprocess" type="preprocess" name="preprocess" args="input:=/rslidar_points" output="screen">
		<param name="on_decription" value="$(arg on_decription)"/>
		<param name="fixed_frame_name" value="$(arg fixed_frame_name)"/>
		<param name="voxel_leafsize" value="$(arg voxel_leafsize)"/>
		<param name="x_max" value="$(arg x_max)"/>
		<param name="x_min" value="$(arg x_min)"/>
		<param name="y_max" value="$(arg y_max)"/>
		<param name="y_min" value="$(arg y_min)"/>
		<param name="z_max" value="$(arg z_max)"/>
		<param name="z_min" value="$(arg z_min)"/>
	</node>
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find pothole)/rviz/pothole.rviz" />
</launch>

